.travis.yml

```diff
@@ -4,33 +4,51 @@
 language: generic
 
 os: osx
-osx_image: beta-xcode6.1
+osx_image: xcode6.4
 
 env:
   matrix:
     
     - CONDA_PY=27
-    - CONDA_PY=34
     - CONDA_PY=35
```

Comment:
I guess since it depends on this: https://github.com/conda-forge/python-utils-feedstock/pull/2
which is in the pipeline.